conn.go

```diff
@@ -110,39 +110,37 @@ type CloseError struct {
 }
 
 func (e *CloseError) Error() string {
```

Comment:
I'm going to bet that this was an optimization for early Go versions with reallocations on each append of the string?

Probably not called enough to warrant a benchmark to confirm.

Reply:
Nevermind, they don't over-allocate either, so it wouldn't change behaviour at all.